### PR TITLE
Enable Container Development

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,19 @@
-FROM python:3.10
+FROM python:3.10 AS base
```
Comment (steveoh) [inline Dockerfile:1]: **thought**: Can this run on python 3.12 or 3.13 which are still getting support?

https://devguide.python.org/versions/

**Review by steveoh — APPROVED** [review]
this is super cool and will come in handy for other projects. Are you happy with how it is functioning?

File: Dockerfile (hunk 1 above)
Comment (stdavis) [reply Dockerfile:1]: I don't remember the context of this project's python version number. I created a new issue for upgrading it.

**Comment by stdavis** [comment]
> Are you happy with how it is functioning?

I haven't spent a lot of time with it yet. But it is promising. I think that there will likely be more refinements in the future such as the extensions that are auto-installed, etc.

**Pull request merged**